packages/libllvm/update-libcompiler-rt.alpm.script

```diff
@@ -0,0 +1,13 @@
+#!@TERMUX_PREFIX@/bin/bash
+
+set -e
+
+RT_PATH="@TERMUX_PREFIX@/lib/clang/@LLVM_MAJOR_VERSION@/lib/linux"
+
+sort -r | while read -r lib; do
```

Comment:
What's happening here?
We're sorting *something* in reverse order, and then link for remove stuff based on that.
But there doesn't seem to be anything that's getting passed into the `while` loop.

Reply:
To understand what's going on here, you need to understand how aplm-hook works. The loop with read is used here to get a list of targets that triggered alpm-hook, and then process them one by one. If you examine aplm-hook in the [`libcompiler-rt`](https://github.com/termux/termux-packages/blob/fix-alpm-hook/packages/libllvm/update-libcompiler-rt.alpm.hook) subpackage, you'll notice the `NeedsTargets` argument, which gives the command to pass targets - [documentation](https://man.archlinux.org/man/alpm-hooks.5.en#ACTIONS).